src/Sparrow/Json/BlittableJsonDocumentBuilder.cs

```diff
@@ -170,6 +170,8 @@ public void Dispose()
 
             _writer.Dispose();
             GlobalCache.Free(_cacheItem);
+            GC.SuppressFinalize(this);
```

Comment:
Does `BlittableJsonDocumentBuilder` need a finalizer?

Reply:
it doesn't have one, removing this